infrastructure/general/setup-full.md

```diff
@@ -0,0 +1,366 @@
+    server, you can issue the following command:
+
+    ```bash
+    sudo apt-mark hold polkadot
+    ```
+
+    ### RPM-based (Fedora, CentOS)
+
+    Currently supports Fedora 32 and CentOS 8, and derivatives.
+
+    ```bash
+    # Install dnf-plugins-core (This might already be installed)
+    dnf install dnf-plugins-core
+    # Add the repository and enable it
+    dnf config-manager --add-repo https://releases.parity.io/rpm/polkadot.repo
+    dnf config-manager --set-enabled polkadot
+    # Install polkadot (You may have to confirm the import of the GPG key, which
+    # should have the following fingerprint: 9D4B2B6EB8F97156D19669A9FF0812D491B96798)
+    dnf install polkadot
+    ```
+
+    !!!note
+        If you choose to use a custom folder for the polkadot home by passing `--base-path '/custom-path'`,
+        you will need to issue following command:
+
+        ```bash
+        sudo mkdir /etc/systemd/system/polkadot.service.d
+        ```
+
+        And create a new file inside this folder:
+
+        ```bash
+        sudo -e /etc/systemd/system/polkadot.service.d/custom.conf
+        ```
+
+        With the following content:
+
+        ```
+        [Service]
+        ReadWritePaths=/custom-path
+        ```
+
+        And finally issue a reload to have your modifications applied by systemd:
+
+        ```bash
+        systemctl daemon-reload
+        ```
+
+## Clone and Build
+
+The [`paritytech/polkadot`](https://github.com/paritytech/polkadot-sdk/tree/master/polkadot) repository
+master branch contains the latest Polkadot code. To install it as a binary, `cargo` may be used to install directly from a specific release:
+
+<!-- TODO: better instructions for installing polka binary -->
+
+```bash
+git clone https://github.com/paritytech/polkadot-sdk polkadot-sdk
+cd polkadot-sdk
+./scripts/init.sh
+cargo build --release
+```
+
+Alternatively, if you wish to use a specific release, you can check out a specific tag (`v0.8.3` in
+the example below):
+
+```bash
+git clone https://github.com/paritytech/polkadot-sdk polkadot-sdk
+cd polkadot-sdk
+git checkout tags/polkadot-v1.9.0
+./scripts/init.sh
+cargo build --release
+```
+
+## Run
+
+The built binary will be in the `target/release` folder, called `polkadot`.
+
+**Polkadot**:
+
+```bash
+polkadot --name "Your Node's Name"
+```
+
+<!-- TODO: link wss tutorial -->
+
+Use the `--help` flag to determine which flags you can use when running the node. For example, if connecting to your node remotely, you'll probably want to use `--rpc-external` and `--rpc-cors all`.
+
+The syncing process will take a while, depending on your capacity, processing power, disk speed and
+RAM. On a $10 DigitalOcean droplet, the process can complete in some ~36 hours.
+
+Congratulations, you're now syncing with Polkadot. Keep in mind that the process is identical when
+using any other Polkadot SDK-based chain.
+
+## Running an Archive Node
```

Comment:
This section seems out of place

Reply:
Added it as a subsection - not sure if we want to move it into its own page? @eshaben

Reply:
These also need to be checked, cause last I checked it's now two separate flags: `state-pruning` and `block-pruning`?

Reply:
You could just put it in one section and use tabs:

```

The built binary will be in the `target/release` folder, called `polkadot`. As a starting point, you can use the following command to run either a full or archive node:

=== "Full"

    ```bash
    polkadot --name "Your Node's Name"
    ```

    !!! note
         The syncing process will take some time, depending on your capacity, processing power, disk speed, and RAM. On a $10 DigitalOcean droplet, it can take about 36 hours to complete.

=== "Archive"

    ```bash
    polkadot --name "My node's name" --pruning archive
    ```

    !!! note
        Using an additional flag, `--wasm-execution compiled`, you can almost quadruple synchronization speed. However, this uses much more CPU and RAM, so it should be turned off after the node syncs.

Use the `--help` flag to determine which flags you can use when running the node. For example, if you want to connect to your node remotely, you'll probably want to use `--rpc-external` and `--rpc-cors all`.

Congratulations, you're now syncing with Polkadot. Keep in mind that the process is identical when
using any other Polkadot SDK-based chain.
```